lib/Entity/Schedule.php

```diff
@@ -810,6 +845,11 @@ private function generateMonth($generateFromDt, $start, $end)
             /** @var Date $start */
             $start = $lastWatermark->copy();
             $end = $start->copy()->addSeconds($eventDuration);
+
+            if (($start <= $generateToDt) && ($end >= $generateFromDt)) {
```

Comment:
This might be clearer and more consistent with the rest of the code if it had fewer brackets?

```
if ($start <= $generateToDt && $end >= $generateFromDt) {
```